lib/inets/src/inets_app/inets.erl

```diff
@@ -47,11 +47,12 @@
 %% Description: Starts the inets application. Default type
 %% is temporary. see application(3)
 %%--------------------------------------------------------------------
-start() -> 
-    application:start(inets).
+start() ->
+    application:ensure_all_started([inets, ssl]).
 
-start(Type) -> 
-    application:start(inets, Type).
+start(Type) ->
+    application:start(inets, Type),
+    application:ensure_all_started(ssl).
```

Comment:
We realized application:ensure_all_started has a different return value that application:start and as we documented the application start return value for ssl:start we need to make it backwards compatible by only returning ok and not {ok, Value}  here.